plugins/redfish/fu-redfish-device.c

```diff
@@ -498,12 +517,23 @@ fu_redfish_device_probe(FuDevice *dev, GError **error)
 			if (json_object_has_member(related_item, "@odata.id")) {
 				const gchar *id =
 				    json_object_get_string_member(related_item, "@odata.id");
-				if (!fu_redfish_device_probe_related_item(self, id, error))
-					return FALSE;
+				g_autoptr(GError) error_local = NULL;
+				if (!fu_redfish_device_probe_related_item(self, id, &error_local)) {
+					if (!g_error_matches(error_local,
+							     FWUPD_ERROR,
+							     FWUPD_ERROR_NOT_SUPPORTED)) {
+						*error = g_error_copy(error_local);
```

Comment:
rather, please do (the slightly insane) `g_propagate_error(error, g_steal_pointer(&error_local))` -- then it doesn't explode if @error is ever NULL.